webui/react/src/components/InteractiveTable.module.scss

```diff
@@ -81,7 +90,17 @@
 }
 .cell {
   .rightClickableCellWrapper {
-    display: inline;
+    display: inline-table;
+    overflow-x: auto;
+    overflow-y: hidden;
+    width: 100%;
+  }
+  /* stylelint-disable at-rule-empty-line-before */
+  // only for firefox
+  @-moz-document url-prefix() {
```

Comment:
Firefox doesn't work like Chrome or Safari, so I did a special treat for Firefox.